tools/fuchsia/debug_symbols.gni

```diff
@@ -0,0 +1,82 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# The inputs to this template are 'binary_path' and a boolean 'unstripped'.
+# If 'unstripped' is specified, we append '.debug' to the symbols name.
+template("_copy_debug_symbols") {
+  assert(defined(invoker.binary_path), "'binary_path' needs to be defined.")
+
+  bin_path = invoker.binary_path
+
+  _args = []
+  if (defined(invoker.unstripped) && invoker.unstripped) {
```

Comment:
I'd generally about optional invoker arguments. Especially in this case where the template is private.

Reply:
Good call.